src/main/java/Todo.java

```diff
@@ -0,0 +1,14 @@
+public class Todo extends Task{
```

Comment:
Same as before, you can add a space after the "Task" and before "{" to comply with coding practice